nixpkgs/default.nix

```diff
@@ -0,0 +1,19 @@
+# How to update this?
```

Comment:
Maybe a bit less wordy. "How to update nix:"

Reply:
Done ✅